core/src/main/resources/i18n/displayStrings.properties

```
@@ -1398,7 +1398,7 @@ account.menu.notifications=Notifications
 
 account.menu.walletInfo.balance.headLine=Wallet balances
 account.menu.walletInfo.balance.info=This shows the internal wallet balance including unconfirmed transactions.\n\
-  For Bitcoin the sum of the 'available balance' and the 'reserved for offers balance' must match the internal wallet balance \
+  For Bitcoin the sum of the 'available balance' and the 'reserved in offers balance' matches the internal wallet balance \
```

More clear to reference "BTC" instead of "Bitcoin" since balance below is labeled as "BTC".

Also label in top right is just labeled "Reserved" so it's probably better to use that string instead of "Reserved in offers" (I know it shows on hover but it's not immediately obvious / most people probably don't hover).

```suggestion
  For BTC, the internal wallet balance shown below should match the sum of the 'Available' and 'Reserved' balances shown in the top right of this window. \
```